src/utils/disks/calculateVDiskSeverity.ts

```diff
@@ -52,5 +52,10 @@ function getColorSeverity(color?: EFlag) {
         return NOT_AVAILABLE_SEVERITY;
     }
 
+    // Blue is reserved for not replicated VDisks
```

Comment:
`getColorSeverity` is used to convert `DiskSpace` and `FrontQueues` flags to severity